app/helpers/most_focus_helper.rb

```diff
@@ -26,9 +26,10 @@ def viewable?(item)
   def get_data_type(model_object, data_type)
     data = []
     model_object.select do |item|
-      next unless item[data_type].any? && viewable?(item)
+      objs = data_type == 'moods' ? item.moods.pluck(:id) : item[data_type]
```

Comment:
We should eventually extract this into helper and generalize his for categories and strategies. This can be handled in a subsequent PR.